Extractor/FrontendExtractor.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Bazinga\Bundle\JsTranslationBundle\Extractor;
+
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Translation\Extractor\AbstractFileExtractor;
+use Symfony\Component\Translation\MessageCatalogue;
+use Symfony\Component\Translation\Extractor\ExtractorInterface;
+
+final class FrontendExtractor extends AbstractFileExtractor implements ExtractorInterface
+{
+    const FUNCTION_STRING_ARGUMENT_REGEX = '\s?[\'"]([^"\'),]+)[\'"]\s?';
+    const REGEX_DELIMITER = '/';
+
+    private $prefix = '';
+
+    private $filesystem;
+
+    private $configuration;
+
+    public function __construct(Filesystem $filesystem, array $configuration = []) {
```

Comment:
rather than an array of configurations here, with each configuration being a non-validated struct expected to have `extensions` and `sequence` key, I would rather have this extractor getting 2 constructor arguments: an array of extensions and a sequence. And then, if you need multiple sequences for different extensions, it would be a matter of registering several services using this class.

Reply:
OK